src/Controls/docs/Microsoft.Maui.Controls/InputView.xml

```diff
@@ -97,46 +97,6 @@
         <remarks>To be added.</remarks>
       </Docs>
     </Member>
-    <Member MemberName="IsSpellCheckEnabled">
```

Comment:
These comments were moved to be inline XML (since that's the way we're documenting now)